### PR TITLE
Improved 5.6.3 release notes

### PR DESCRIPTION
**Pull request by robbavey**

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -11,9 +11,12 @@ This section summarizes the changes in the following releases:
 [[logstash-5-6-3]]
 === Logstash 5.6.3 Release Notes
 
+* Known Issue: Running Logstash with log level set to debug will cause the pipeline to crash
+* Improved Pipeline performance
```
Comment (original-brownbear) [inline docs/static/releasenotes.asciidoc]: @robbavey probably don't want to capitalize "Pipeline" here if we didn't also capitalize it in the above point?

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -11,9 +11,12 @@ This section summarizes the changes in the following releases:
...
 * Fixed issue where DLQ would crash at startup when a segment file was missing
 * Fixed issues with proxy handling when installing plugins
 
+
```
Comment (original-brownbear) [inline docs/static/releasenotes.asciidoc]: @robbavey accidental empty line? :)

**Comment by robbavey** [comment]
@original-brownbear Thanks - I've addressed your comments

**Comment by original-brownbear** [comment]
LGTM

**Pull request closed without merging**